Seq.Client.Portable/Client/Portable/PortableSeqSink.cs

```diff
@@ -35,17 +36,18 @@ class PortableSeqSink : PortablePeriodicBatchingSink
         public const int DefaultBatchPostingLimit = 1000;
         public static readonly TimeSpan DefaultPeriod = TimeSpan.FromSeconds(2);
 
-        public PortableSeqSink(string serverUrl, string apiKey, int batchPostingLimit, TimeSpan period)
+        public PortableSeqSink(string serverUrl, string apiKey, int batchPostingLimit, TimeSpan period, IHttpClientFactory httpClientFactory)
             : base(batchPostingLimit, period)
         {
             if (serverUrl == null) throw new ArgumentNullException("serverUrl");
             _apiKey = apiKey;
+            _httpClientFactory = httpClientFactory;
 
             var baseUri = serverUrl;
             if (!baseUri.EndsWith("/"))
                 baseUri += "/";
 
-            _httpClient = new HttpClient { BaseAddress = new Uri(baseUri) };
+            _httpClient = _httpClientFactory.CreateHttpClient(new Uri(baseUri));
```

Comment:
Instead of passing the base address through, is it possible to get rid of the `baseUri` argument and just do: `_httpClient.BaseAddress = new Uri(baseUri)` on the returned instance? I don't think there's much that an `IHttpClientFactory` implementation could do with `baseUri` in this case anyway. Thoughts?